nifi-extension-bundles/nifi-opentelemetry-bundle/nifi-opentelemetry-processors/src/main/java/org/apache/nifi/processors/opentelemetry/ListenOTLP.java

```diff
@@ -85,6 +90,7 @@ public class ListenOTLP extends AbstractProcessor {
             .description("TCP port number on which to listen for OTLP Export Service Requests over HTTP and gRPC")
             .required(true)
             .defaultValue("4317")
+            .identifiesListenPort(TransportProtocol.TCP, "http/1.1", "h2", "grpc", "otlp")
```

Comment:
Although minor, what do you think about defining static variables for these protocols and reusing in the `getListenPorts()` method?